frontend/src/app/supportAdmin/TenantDataAccess/operations.graphql

```diff
@@ -1,5 +1,6 @@
 query GetOrganizations($identityVerified: Boolean) {
     organizations(identityVerified: $identityVerified) {
+        internalId
```

Comment:
:warning the `GetOrganizationsWithFacilities` query requries the org's `internalId` as a param